client-misk/src/main/kotlin/app/cash/backfila/client/misk/Backfill.kt

```diff
@@ -85,3 +85,10 @@ abstract class Backfill<E : DbEntity<E>, Pkey : Any, Param : Any> {
  * Placeholder class to represent when a backfill has no Parameters.
  */
 class NoParameters private fun constructor() { /* Never create this */ }
+
+/**
+ * Annotation used on your backfill class and parameters class constructor to send metadata to Backfila.
+ */
+@Target(AnnotationTarget.VALUE_PARAMETER, AnnotationTarget.CLASS)
```

Comment:
how about a file for this

Reply:
I thought that having it part of the Backfill.kt file made sense. I can make it a separate file though.

Reply:
Starting to feel cluttered I guess

Reply:
You like this better?

Reply:
looks good